components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -697,6 +697,21 @@ public static int getIndex(String order,
     };
   }
 
+  public static int[] getXYZCTIndexes(String order, int[] values) {
```

Comment:
Is this meant to return in `XYZCT` order (as the name suggests) or `XYZTC` order (as implemented)?

Reply:
Good catch, updated now to match the name